src/main/kotlin/net/maizegenetics/phgv2/pathing/QcReadMapping.kt

```diff
@@ -0,0 +1,114 @@
+package net.maizegenetics.phgv2.pathing
```

Comment:
Shouldn't this be in phgv2.cli?

Reply:
No, all the other path finding Clikt commands are in their own package so not everything is in the cli package.  This is related to pathing so I put it in there.